src/constructs/ec2/security-groups/base.ts

```diff
@@ -78,17 +89,19 @@ export class GuPublicInternetAccessSecurityGroup extends GuSecurityGroup {
   }
 }
 
+// TODO should this be a singleton?
 export class GuHttpsEgressSecurityGroup extends GuSecurityGroup {
-  constructor(scope: GuStack, id: string, props: SecurityGroupProps) {
+  constructor(scope: GuStack, id: string, props: GuSecurityGroupProps) {
     super(scope, id, {
-      vpc: props.vpc,
+      ...props,
       allowAllOutbound: false,
       description: "Allow all outbound HTTPS traffic",
+      ingresses: [],
```

Comment:
This has the effect of removing any ingresses provided on the `props`. Ideally the interface of `props` should omit this field; I'm planning that in a future bit of work!